dnf-behave-tests/features/vars-repo_id.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Subtitute variables in repo id
```

Comment:
There are already two feature files called `vars-*`. At first glance, one is non-trivial with some background setup, the other has one test and no background. Could we not add another feature file with a couple of tests and instead somehow group them so that there's a balance between the number of files and the size/complexity of their content?

Reply:
OK. Nové testy jsem přesunul do existujícího souboru `vars-mirrorlist.feature `a přejmenoval jsem ho na `vars.feature`.

Reply:
Cheers :slightly_smiling_face: